src/backend/config/app.config.ts

```diff
@@ -0,0 +1,65 @@
+import {IsInt, IsNotEmpty, IsString, IsUrl, Max, Min, validateSync, ValidationError} from "class-validator"
+import {plainToInstance} from "class-transformer";
+
+export class AppConfig {
+  @IsUrl({
+    protocols: ["http", "https"],
+    requireProtocol: true,
+    requireHost: true,
+    require_tld: false,
+  })
+  VITE_BACKEND_URL?: string;
+  
+  @IsString()
+  @IsNotEmpty()
+  DATABASE_CONNECTOR?: string;
+
+  @IsString()
+  @IsNotEmpty()
+  DATABASE_USER?: string;
+
+  @IsString()
+  @IsNotEmpty()
+  DATABASE_PASSWORD?: string;
+
+  @IsInt()
+  @Min(1)
+  @Max(65535)
```

Comment:
I think @IsInt should be at the bottom. Decorators are invoked from bottom to top.